source/support.tex

```diff
@@ -750,6 +750,7 @@
 #define @\defnlibxname{cpp_lib_unreachable}@                       202202L // also in \libheader{utility}
 #define @\defnlibxname{cpp_lib_unwrap_ref}@                        201811L // also in \libheader{type_traits}
 #define @\defnlibxname{cpp_lib_variant}@                           202106L // also in \libheader{variant}
+#define @\defnlibxname{cpp_lib_within_lifetime}@                   202306L // also in \libheader{type_traits}
```

Comment:
Whyever did they exclude `is_`?
There's plenty of `<type_traits>` macros with it,
including that of its neighbour `is_constant_evaluated`.

Reply:
We can fix that.